tests/core/test_arg_parser.py

```
@@ -0,0 +1,153 @@
+    """Test suite for --modules-extra-args parsing"""
+
+    @patch("nettacker.core.arg_parser.ArgParser.load_graphs")
+    @patch("nettacker.core.arg_parser.ArgParser.load_languages")
+    @patch("nettacker.core.arg_parser.ArgParser.load_modules")
+    @patch("nettacker.core.arg_parser.ArgParser.load_profiles")
+    @patch("nettacker.core.arg_parser.ArgParser.add_arguments")
+    @patch("sys.exit")
+    def test_modules_extra_args_without_equal_sign(
+        self,
+        mock_exit,
+        mock_add_args,
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_add_args`

(ARG002)